README.md

```diff
@@ -1,2 +1,141 @@
+
+# 📖 Portfolio <a name="about-project"></a>
+
+**Portfolio** is a project to is a project to show my skills and showcase the projects I have done.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.spec.whatwg.org/">HTML</a></li>
+    <li><a href="https://www.w3.org/TR/CSS/#css">CSS</a></li>
+    <li><a href="https://www.javascript.com/">Javascript</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Display services available at the clinic]**
+- **[Display timetable and opening hours]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+Hardware
+
+1. at leats 4 GB RAM
+2. Internal disk space of at least 2 GB
+
+Software
+
+1. Web Browser
+2. Git
+3. Code editor
+
+### Setup
+
+Clone this repository to your desired folder Enter the following commands in the terminal:
+
+```sh
+  git clone https://github.com/FilbertKimasa/Mobile-clinic.git
+```
+
+### Install
+
+Install this project with the following commmands in the terminal:
+
+```sh
+  1. npm init -y
+  2. cd my-project
+  3. npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+  4. npm install --save-dev hint@7.x
+  5. npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
+```
+
+### Usage
+
+To run the project,
+
+```sh
+  Open the browser and enter the adress of index.html file from your computer/server
+```
+
+### Run tests
+
+```sh
+   1. npx hint .
+   2. npx stylelint "**/*.{css,scss}"
+   3. npx eslint .
+```
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Filbert Daniel**
+
+- GitHub: [@githubhandle](https://github.com/FilbertKimasa)
+- Twitter: [@twitterhandle](https://twitter.com/filbertdan67)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/filbert-daniel-32b118143)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add some more HTML content**
+- [ ] **Intergration with javascript**
+- [ ] **Intergration with database**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 contributing <a name="contributing"></a>
+
+You are welcome, and click <a href="https://github.com/FilbertKimasa/Mobile-clinic/issues">here</a> for any issues🙏
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give it a star and follow me on github linkedIn and Twitter
+, you are also welcome for any suggestions🙏
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank my Microverse studying partners and my standup team for helping and encouraging me to do this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Please give credits to the design owner for a professional repo. 🙏 [check this](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#:~:text=The%20Creative%20Commons%20license%20of%20the%20design%20requires%20that%20you%20give%20appropriate%20credit%20to%20the%20author.%20Therefore%2C%20you%20must%20do%20it%20in%20the%20README%20of%20your%20project.)